xbmc/cores/DllLoader/dll_tracker.cpp

```diff
@@ -67,7 +67,7 @@ void tracker_dll_free(DllLoader* pDll)
   }
 }
 
-void tracker_dll_set_addr(DllLoader* pDll, uintptr_t min, uintptr_t max)
+void tracker_dll_set_addr(const DllLoader* pDll, uintptr_t min, uintptr_t max)
```

Comment:
i think this still allows `pDll = PointerToSomeOtherDllOrNullptr` - should it be `const DllLoader* const pDll`

Reply:
The same argument goes for `uintptr_t min = 0`, we generally don't use `const` on function parameters, because the "outside" isn't affected by changes to those parameters